iwyu_globals.cc

```diff
@@ -296,16 +296,14 @@ static vector<HeaderSearchPath> NormalizeHeaderSearchPaths(
 static vector<HeaderSearchPath> ComputeHeaderSearchPaths(
     clang::HeaderSearch* header_search) {
   map<string, HeaderSearchPath::Type> search_path_map;
-  for (clang::HeaderSearch::search_dir_iterator
-           it = header_search->system_dir_begin();
+  for (auto it = header_search->system_dir_begin();
```

Comment:
This is one of the few places where I prefer auto -- where the iterator type name is so long that it dwarfs everything else :-). This, and when the type is clearly visible on the right-hand side.